src/cirrus/builtins/cloudformation/resources.yml

```diff
@@ -61,6 +61,20 @@ Resources:
             ProjectionType: ALL
       BillingMode: PAY_PER_REQUEST
       TableName: "#{AWS::StackName}-state"
+  StateEventTimestreamDatabase:
+    Type: AWS::Timestream::Database
+    Properties:
+      DatabaseName: "#{AWS::StackName}-state-events"
+      # KmsKeyId: String -- use default one
+  StateEventTimestreamTable:
+    Type: AWS::Timestream::Table
+    DependsOn: StateEventTimestreamDatabase
+    Properties:
+      TableName: "#{AWS::StackName}-state-events-table"
+      DatabaseName: "#{AWS::StackName}-state-events"
+      RetentionProperties:
+        MemoryStoreRetentionPeriodInHours: "24"
+        MagneticStoreRetentionPeriodInDays: "93"
```

Comment:
@jkeifer changed this to 93 from 3650 (10 years), for at least 2 full months of data. Keeping this around forever is probably unnecessary, and users can change it if they want.